bin/bs-bench.scala

```diff
@@ -56,33 +54,38 @@ object Main
         {
             if (varyPlots)
             {
-                // As long as oldPath is 6.3 we can't use --update-plots
-                // so at some point we could test the version number
-                // to see if it is < 6.4  aab 10-29-2023
-                // data = data :+ time(oldPath, true)
-                data = data :+ time(oldPath, false)
+                try data = data :+ time(newPath, true) catch case _ => {}
+                try data = data :+ time(newPath, false) catch case _ => {}
             }
 
-            else data = data :+ time(oldPath, updatePlots)
+            else try data = data :+ time(oldPath, updatePlots) catch case _ => {}
         }
 
         if (varyPlots)
         {
-            data = data :+ time(newPath, true)
-            data = data :+ time(newPath, false)
+            try data = data :+ time(newPath, true) catch case _ => {}
+            try data = data :+ time(newPath, false) catch case _ => {}
         }
 
-        else data = data :+ time(newPath, updatePlots)
+        else try data = data :+ time(newPath, updatePlots) catch case _ => {}
 
         if (!outputFile.isEmpty)
         {
             val output = new java.io.PrintWriter(new java.io.File(outputFile))
 
             output.write("Name,Update Plots")
 
-            for (i <- 1 to trials) output.write(s",Trial $i")
+            if (trials > 1)
+            {
+                for (i <- 1 to trials) output.write(s",Trial $i")
+
+                output.write(",Average,Standard Deviation\n")
```

Comment:
ideally avg and std dev come before the data for each trial
currently output is

```
Name,Update Plots,Trial 1,Trial 2,Trial 3,Average,Standard Deviation
NetLogo 6.3.0,false,0.1292611,0.101793416,0.09748176,0.10951209,0.014075161342616912
NetLogo 6.4.0-beta1,true,0.12559584,0.100356445,0.10418547,0.11004592,0.0111060159612062
NetLogo 6.4.0-beta1,false,0.10071372,0.10369839,0.1017232,0.1020451,0.0012395625204687849
```